src/pages/Landing/components/Body/FileBox/FileBox.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable import/no-unresolved */
+import React from 'react'
+import groupIcon from 'assets/images/group-large.png'
+import arrowUpIcon from 'assets/images/arrow-up.png'
+import styles from './FileBox.module.scss'
+import Item from './Item/Item'
+
+const files = [
+  {
+    isFile: true,
+    name: 'References.zip',
+    date: 'Oct 20, 2020 at 21:00',
+    size: '1.2MB',
+  },
+  {
+    isFile: true,
+    name: 'References.zip',
+    date: 'Oct 20, 2020 at 21:00',
+    size: '1.2MB',
+  },
+  {
+    isFile: true,
+    name: 'References.zip',
+    date: 'Oct 20, 2020 at 21:00',
+    size: '1.2MB',
+  },
+]
+
+const links = [
+  {
+    isFile: false,
+    name: 'Muz.li',
+    date: 'Oct 20, 2020 at 21:00',
+    size: '1.2MB',
+  },
+  {
+    isFile: false,
+    name: 'Muz.li',
+    date: 'Oct 20, 2020 at 21:00',
+    size: '1.2MB',
+  },
+  {
+    isFile: false,
+    name: 'Muz.li',
+    date: 'Oct 20, 2020 at 21:00',
+    size: '1.2MB',
+  },
+]
```

Comment:
can you please take out dumb data in a separate files. 
try to keep component logic and design separately (if it is necessary, then use custom hooks)